### PR TITLE
Added Patient and Prescriptions details

### PR DESCRIPTION
**Pull request by Utkarsh-81**
Added Patient and Prescriptions details, added Html and CSS for layout

### CODE REVIEW

File: app/component/EmailUs.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import styles from "./css/EmailUs.module.css";
+
+const EmailUs = () => {
+  return (
+    <>
+      <div>
+        <button
+          style={{ width: "8vw" }}
```
Comment (pratheesh-repo) [inline app/component/EmailUs.tsx:8]: Inline css is not recomendend.

File: app/component/FaxUs.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import styles from "./css/FaxUs.module.css";
+
+const FaxUs = () => {
+  return (
+    <>
+      <div>
+        <button
+          style={{ width: "8vw" }}
```
Comment (pratheesh-repo) [inline app/component/FaxUs.tsx:8]: Inline css is not recomendend.

File: app/component/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+            </div>
+            <div className="col-md-3 col-lg-2 col-xl-2 mx-auto mt-3">
+              <h5 className="text-uppercase mb-4 font-weight-bold text-warning">
+                Useful Links
+              </h5>
+              <p>
+                <a
+                  href="#"
+                  className="text-white"
+                  style={{ textDecoration: "none" }}
```
Comment (pratheesh-repo) [inline app/component/Footer.tsx:68]: avoid Inline css.

File: app/component/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+                >
+                  Your Account
+                </a>
+              </p>
+              <p>
+                <a
+                  href="#"
+                  className="text-white"
+                  style={{ textDecoration: "none" }}
```
Comment (pratheesh-repo) [inline app/component/Footer.tsx:77]: avoid Inline css.

File: app/component/Search.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  const [value, setValue] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://jsonplaceholder.typicode.com/posts")
+      .then((res) => {
+        setValue(res.data);
+      })
+      .catch((err) => {
+        console.log(err);
```
Comment (pratheesh-repo) [inline app/component/Search.tsx:17]: use pino for logging.

File: app/pages/api/hello.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```
Comment (pratheesh-repo) [inline app/pages/api/hello.ts:1]: This file can be deleted

File: app/pages/prescriptions/index.tsx [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+      `,
+    });
+
+    return {
+      props: {
+        data: data.posts.data,
+      },
+    };
+  } catch (error) {
+    logger.info(`Error: ${error}`);
```
Comment (pratheesh-repo) [inline app/pages/prescriptions/index.tsx:43]: use logger.error(\`Error: ${error}\`);

**Review by pratheesh-repo — COMMENTED** [review]
added few comments

File: app/component/EmailUs.tsx (hunk 1 above)
Comment (Utkarsh-81) [reply app/component/EmailUs.tsx]: Addressed

File: app/component/FaxUs.tsx (hunk 2 above)
Comment (Utkarsh-81) [reply app/component/FaxUs.tsx]: Addressed

File: app/component/Footer.tsx (hunk 3 above)
Comment (Utkarsh-81) [reply app/component/Footer.tsx]: Addressed

File: app/component/Footer.tsx (hunk 4 above)
Comment (Utkarsh-81) [reply app/component/Footer.tsx]: Addressed

File: app/pages/prescriptions/index.tsx (hunk 7 above)
Comment (Utkarsh-81) [reply app/pages/prescriptions/index.tsx:43]: Addressed

File: app/pages/api/hello.ts (hunk 6 above)
Comment (Utkarsh-81) [reply app/pages/api/hello.ts]: Addressed

**Review by pratheesh-repo — APPROVED** [review]

**Pull request merged**